### DISCUSSION TITLE
Need solo without mrc20

### DISCUSSION THREAD

**Discussion by ViperStoner** [community/community#123180] [Code Security] [answered]
### Select Topic Area

Question

### Body

Can i get solo contract without mrc

**Comment by techpond31** [comment, marked as answer]
🤔No, it is not possible to acquire an Xbox Series S All Access Contract without paying Monthly Reoccurring Costs (MRC). The MRC includes the cost of the console plus Xbox Game Pass Ultimate for 24 monthly payments. This is the typical financing method for obtaining a console through the All Access program. Without MRC, you would need to buy the Xbox Series S console at retail price without the accompanying benefits of the Game Pass Ultimate membership.
